config/datasets/gcp.yaml

```diff
@@ -43,6 +46,7 @@ catalog_study_locus: ${datasets.study_locus}/catalog_study_locus
 gwas_catalog_study_curation: ${datasets.inputs}/v2d/GWAS_Catalog_study_curation.tsv
 finngen_study_index: ${datasets.study_index}/finngen
 finngen_summary_stats: ${datasets.summary_statistics}/finngen
+finngen_finemapping_out: gs://genetics-portal-dev-analysis/xg1/Finngen_finemapping_r10_processed
```

Comment:
Since you are outputting credible sets, I think this should be written to `${datasets.credible_set}/finngen`